ibm_catalog.json

```diff
@@ -0,0 +1,254 @@
+{
+  "products": [
+    {
+      "label": "Landing zone for cloud-native AI applications",
+      "name": "Retrieval_Augmented_Generation_Pattern",
+      "product_kind": "solution",
+      "tags": [
+        "solution",
+        "watson",
+        "security",
+        "banking",
+        "ibm_created"
+      ],
+      "keywords": [
+        "cloud-native",
+        "rag",
+        "watson",
+        "ai",
+        "compliance",
+        "fscloud",
+        "genai",
+        "sample",
+        "ibmcloud",
+        "financial services",
+        "watsonx",
+        "llm",
+        "retrieval augmented generation",
+        "secure",
+        "secret manager",
+        "key protect",
+        "security and compliance center workload protection",
+        "cspm",
+        "config aggregator",
+        "app config"
+      ],
+      "short_description": "Automate a cloud-native AI deployment with supporting IBM Cloud and watsonx services, embed your enterprise data in generative AI solutions",
+      "long_description": "This solution enables the provisioning of cloud-native application landing zones on IBM Cloud using a range of configurations tailored to different needs — from quick experimentation to enterprise-grade deployments. Each variation offers a distinct balance of automation, integration with IBM Cloud services for security, observability, and compliance, and readiness for evaluation or production use. Whether you're deploying serverless applications on IBM Code Engine or containerized workloads on Red Hat OpenShift, these configurations help accelerate your cloud-native journey. <br/> The Landing zone for cloud-native AI applications includes three core configurations: Quick Start for lightweight experimentation, Basic for flexible serverless deployments on Code Engine with foundational services, and Standard for production-ready OpenShift clusters with advanced security and compliance. Both Basic and Standard also offer variants that include a deployed sample application with watsonx integration and CI/CD pipelines for secure lifecycle management.",
+      "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-cloud-native-ai/blob/main/README.md",
```

Comment:
This value has to be the same in the two repos since both use the same tile. We may wish to create a cloud doc entry for this tile in Q1, but for now, just make sure they have same value